io/include/pcl/io/ply/ply_parser.h

```diff
@@ -383,18 +383,23 @@ namespace pcl
             std::vector<std::shared_ptr<property>> properties;
           };
           
-          info_callback_type info_callback_;
-          warning_callback_type warning_callback_;
-          error_callback_type error_callback_;
+          info_callback_type info_callback_ = [](std::size_t, const std::string&){};
+          warning_callback_type warning_callback_ = [](std::size_t, const std::string&){};
+          error_callback_type error_callback_ = [](std::size_t, const std::string&){};
           
-          magic_callback_type magic_callback_;
-          format_callback_type format_callback_;
-          element_definition_callback_type element_definition_callbacks_;
+          magic_callback_type magic_callback_ = [](){};
+          format_callback_type format_callback_ = [](format_type, const std::string&){};
+          comment_callback_type comment_callback_ = [](const std::string&){};
+          obj_info_callback_type obj_info_callback_ = [](const std::string&){};
+          end_header_callback_type end_header_callback_ = [](){return true;};
+
+          element_definition_callback_type element_definition_callbacks_ = 
+              [](const std::string&, std::size_t)
+              {
+                  return std::make_tuple([](){}, [](){});
```

Comment:
:boom: :laughing:

Reply:
I tried fitting in more brackets but it wouldn't compile 🤣